networks/umee-betanet-2/upgrade.md

```diff
@@ -1,54 +1,57 @@
 <!-- markdownlint-disable MD013 -->
-# Upgrade Instructions
+
+# Betanet-2
+
+## Upgrade Instructions
```

Comment:
This too is deprecated - betanet must be pretty old (`v0.x`) if it still references rinkeby and gorc.

We can remove this network

Reply:
yes, was thinking about it but wasn't sure if we wanted to keep it for some reason.
Removing now.